src/main/kotlin/com/criptext/mail/scenes/settings/aliases/AliasesController.kt

```diff
@@ -378,8 +380,10 @@ class AliasesController(
                                 result.userSettings.aliases
                         )
                 )
-                if(result.userSettings.customerType == AccountTypes.STANDARD){
-                    if(activeAccount.type == AccountTypes.STANDARD){
+                if(result.userSettings.customerType == AccountTypes.STANDARD
+                        || result.userSettings.customerType == AccountTypes.LUCKY){
+                    if(activeAccount.type == AccountTypes.STANDARD
```

Comment:
why double validation? If it's important, why you don't do it in `customDomainController`?